AppConnectSwift/CaptureImageViewController.swift

```diff
@@ -48,12 +90,17 @@ class CaptureImageViewController: UIViewController, UIImagePickerControllerDeleg
     
     @IBAction func saveTapped(sender: AnyObject) {
         if image.CGImage != nil {
-            let data = compressFile() as? NSData
-            var subject: MDSubject!
-            subject.collectData(data, withMetadata: "Random String", completion: { (dataEnvelope: MDSubjectDataEnvelope!, err: NSError!) -> Void in
-                print(err == nil ? "Data Saved" : err?.description);
-                self.imageView.image = nil
-            })
+            for subject in self.collectedSubjects {
```

Comment:
I think for the sample app purposes we'll only want to collectData for a single subject. In most cases there will only be one, but it might make more sense to just store one when the view loads, and then use it here (like you're doing with collectedSubjects)